tests/jenkins/jobs/Build_Jenkinsfile

```diff
@@ -112,6 +118,12 @@ pipeline {
                                 upload: false,
                                 manifest: 'tests/data/opensearch-build-1.1.0.yml'
                             )
+                            uploadTestResults(
+                                upload: false,
+                                manifest: 'tests/data/opensearch-build-1.1.0.yml',
+                                jobName: "bundle_build",
```

Comment:
Should this have been `bundle-build`?

Reply:
This is for S3. We have it there as `bundle_build`.

Reply:
Nope its `bundle-build` even in S3. There is also `bundle_build` which is incorrect and contains just 1 build

Reply:
I fixed it in https://github.com/opensearch-project/opensearch-build/pull/1111 if you care to CR.

Reply:
Missed that we have 2 buckets. Thanks @dblock for the fix.

Reply:
@gaiksaya I think it's safe to delete `bundle_build` in S3 to avoid confusion - want to do it?

Reply:
Yes and few others which were created based on job-names :) Will delete the unnecessary ones